rclcpp/include/rclcpp/node.hpp

```diff
@@ -142,76 +142,27 @@ class Node : public std::enable_shared_from_this<Node>
   /// Create and return a Publisher.
   /**
    * \param[in] topic_name The topic for this publisher to publish on.
-   * \param[in] qos_history_depth The depth of the publisher message queue.
```

Comment:
Reminder: we need to add back in the pre-existing `create_publisher` and `create_subscriber` APIs, and to mark them as deprecated